gitpod-io/gitpod
# [supervisor] inflate all git repos

fixes [multi-repo] Shallow clones are not inflated #9021

## How to test

start a workspace on a multi-repo. E.g. https://sefftinge-362778b072.preview.gitpod-dev.com/#https://github.com/svenefftinge/multi-repo-main
And verify that the environment variable `GITPOD_REPO_ROOTS` contains all three locations (comma separated) and all repositories are fully cloned (check with `git log`).

```release-notes
NONE
```
